apps/web/components/ProductAccordion/ProductAccordion.vue

```diff
@@ -9,31 +9,41 @@
           {{ $t('productDetails') }}
         </h2>
       </template>
-      <p>{{ product.description }}</p>
+      <p>{{ productGetters.getDescription(product) }}</p>
     </UiAccordionItem>
     <UiDivider class="my-4" />
     <UiAccordionItem
+      v-if="productReviews && (reviewGetters.getItems(productReviews)?.length ?? 0 > 0)"
```

Comment:
Looks strange to me, would this not work?
`productReviews && reviewGetters.getItems(productReviews)?.length > 0`